src/lib/vocab/entity/entityRepo.test.ts

```diff
@@ -0,0 +1,32 @@
+import {suite} from 'uvu';
+import * as assert from 'uvu/assert';
+
+import {setupDb, teardownDb, type TestDbContext} from '$lib/util/testDbHelpers';
+import {toRandomVocabContext} from '$lib/vocab/random';
+import type {TestAppContext} from '$lib/util/testAppHelpers';
+
+/* test__entityRepo */
+const test__entityRepo = suite<TestDbContext & TestAppContext>('entityRepo');
+
+test__entityRepo.before(setupDb);
+test__entityRepo.after(teardownDb);
+
+test__entityRepo('entites return sorted by created', async ({db}) => {
+	const random = toRandomVocabContext(db);
+	const space = await random.space();
+	const entity0 = await random.entity(undefined, undefined, undefined, space);
+	const entity1 = await random.entity(undefined, undefined, undefined, space);
+	const entity2 = await random.entity(undefined, undefined, undefined, space);
+
+	assert.equal(entity0.space_id, entity1.space_id);
+	assert.equal(entity1.space_id, entity2.space_id);
+
```

Comment:
```suggestion

	// Ensure db sort order is shuffled from the insertion order.
	await db.repos.entity.updateEntityData(entity1.entity_id, entity1.data);
```

Looks like the test passes without the new `ORDER BY` change. This makes sure we're not relying on insertion order.